### ISSUE TITLE
Panasonic AW-HE40 showing as black

### ISSUE THREAD

**Issue by amyamazing** [impsnldavid/jit.ndi#13]
OSX 10.14.6 
Max/Msp 8.1.8
Last Newtek NDI HX Driver installed
jit.ndi v0.2.0

Hi,
in this patch, there are:
- a stream from the app NDI HX Camera for iphone.
- a stream generated by NDI Signal Generator.
- a stream from a camera with NDI HX upgrade support. 

![jit ndi](https://user-images.githubusercontent.com/76221741/102751763-a80cc100-4368-11eb-8654-f9be21d8e5a4.jpg)

As you can see this last stream is not processed by jit.ndi. 
(I've tried different resolutions of this stream from 720p to 1080p both PAL 25 & 50fps, and NTSC 30 & 60fps.)

Additionally, trying to locate the problem, I've tested different builds of Millumin having different versions of NDI Libraries installed from 3.5 to 4.6. No issue to report.

Feel free to ask me to try anything,
Thanks

**Comment by pixsperdavid** [comment]
So this appears to be a problem with this specific device rather than anything applicable to all HX devices, as evidenced by the fact the iPhone camera app works. I've never tested the Panasonic PTZ cameras as I've never had access to one with the NDI upgrade installed. Unfortunately to make much progress resolving this I would need to run through the connection process with this device in the debugger.

**Comment by amyamazing** [comment]
Thanks, I understand.

One difference I see between these streams is this weird source ip :
<img width="315" alt="ip" src="https://user-images.githubusercontent.com/76221741/102774566-86710100-438b-11eb-862b-da47b3b22bba.png">
Is there a way I could point the jit.ndi.receive directly to this ip maybe ?
Something like jit.ndi.receive~ 2 @ndi-hx-pana-ptz-v1://192.168.0.11 ?

**Comment by amyamazing** [comment]
Quick follow up, for those who might have the same issue. 

I've installled a runtime linked in the previous issue by LSka https://ndi.palakis.fr/runtime/ndi-runtime-4.5.1-macOS.pkg

Before, jit.ndi was receiving these stream dimensions :
<img width="359" alt="Capture d’écran 2020-12-21 à 17 31 36" src="https://user-images.githubusercontent.com/76221741/102807317-b38ad700-43be-11eb-8308-d08251c1c466.png">
Now, everything seems ok :
<img width="273" alt="Capture d’écran 2020-12-21 à 18 59 29" src="https://user-images.githubusercontent.com/76221741/102807343-c0a7c600-43be-11eb-9174-7eff88838dd4.png">

<img width="1265" alt="Capture d’écran 2020-12-21 à 19 07 53" src="https://user-images.githubusercontent.com/76221741/102809758-d9b27600-43c2-11eb-8ee1-c80182b2b632.png">

By the way, I have no idea what version of libndi.4.dylib was previously in my folder /usr/local/lib/
I've tried to uninstall it & re-install 3 different versions of the NewtekNDITools & HX drivers, no libndi.4.dylib was ever installed.
So, I guess Newtek hasn't automatically provided an NDI runtime with its pkg for a while... btw the libndi.4.dylib inside the last version of the "Newtek NDI Video Monitor" package works fine too.
